MapboxNavigation/NSExpression.swift

```diff
@@ -0,0 +1,80 @@
+import Foundation
+
+extension Dictionary where Key == Float, Value == NSExpression {
+    func localized(into locale: Locale?, replacingTokens replacesTokens: Bool) -> [Float: NSExpression] {
+        var localizedStops = self
+        var hasLocalizedValue = false
+        for (zoomLevel, value) in localizedStops {
+            let localizedValue = value.localized(into: locale, replacingTokens: replacesTokens)
+            if localizedValue != value {
+                localizedStops[zoomLevel] = localizedValue
+                hasLocalizedValue = true
+            }
+        }
+        return hasLocalizedValue ? localizedStops : self
+    }
+}
+
+extension NSExpression {
+    func localized(into locale: Locale?, replacingTokens replacesTokens: Bool) -> NSExpression {
```

Comment:
Nearly all of this functionality has been upstreamed to the map SDK in mapbox/mapbox-gl-native#11651. So the implementation here can be replaced with a simple call to the similarly named method provided by the map SDK.

Reply:
Done in 7bbfe0deb187c350638ff579788ee843bea50474.